testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -345,82 +341,64 @@ def test_Writer_is_LAMMPS(self):
             assert_(W.flavor, self.flavor)
 
     def test_open(self):
-        def open_dcd():
-            try:
-                with mda.coordinates.LAMMPS.DCDWriter(self.outfile, n_atoms=10):
-                    pass
-            except Exception:
-                return False
-            else:
-                return True
-        assert_(open_dcd(), True)
+        try:
+            with mda.coordinates.LAMMPS.DCDWriter(self.outfile, n_atoms=10):
+                pass
+        except Exception:
+            pytest.fail()
```

Comment:
This is a bit awkward, but probably ok; see: https://stackoverflow.com/a/20275035